react/index.js

```diff
@@ -286,7 +286,31 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
-    const style = { visibility: showBuyButton ? 'visible' : 'hidden' }
+    const buyButtonStyle = { visibility: showBuyButton ? 'visible' : 'hidden' }
+
+    const renderBuyButton = () => (
```

Comment:
I think this function should be a getter just like the other render functions of the component